client/e2e/testData/ConstsPackage.ts

```diff
@@ -8,66 +19,161 @@ export interface EventsInfo {
 
 // This is an instance of the object
 export const EventsInfo1: EventsInfo = {
-  eventName: "S",
-  eventDescription: "123",
-  eventURL: "http://",
-  eventFullName: "S Playbill S Description 123"
+  eventName: 'S',
+  eventDescription: '123',
+  eventURL: 'http://',
+  eventFullName: 'S Playbill S Description 123',
 };
 
 export const EventsInfo2: EventsInfo = {
-  eventName: "Test_event",
-  eventDescription: "An event for testing",
-  eventURL: "https://www.hindustantimes.com/ht-img/img/2023/08/25/550x309/international_dog_day_1692974397743_1692974414085.jpg",
-  eventFullName: "Test_event Playbill Test_event Description An event for testing"
+  eventName: 'Test_event',
+  eventDescription: 'An event for testing',
+  eventURL: 'https://www.hindustantimes.com/ht-img/img/2023/08/25/550x309/international_dog_day_1692974397743_1692974414085.jpg',
+  eventFullName: 'Test_event Playbill Test_event Description An event for testing',
 };
 
 export const EventsInfo3: EventsInfo = {
-  eventName: "The Crucible1",
-  eventDescription: "111Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
-  eventURL: "https://www.hindustantimes.com/ht-img/img/2023/08/25/550x309/international_dog_day_1692974397743_1692974414085.jpg",
-  eventFullName: "The Crucible1 Playbill The Crucible1 Description 111Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua."
+  eventName: 'The Crucible1',
+  eventDescription: '111Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.',
+  eventURL: 'https://www.hindustantimes.com/ht-img/img/2023/08/25/550x309/international_dog_day_1692974397743_1692974414085.jpg',
+  eventFullName: 'The Crucible1 Playbill The Crucible1 Description 111Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.',
 };
 
 export const EventsInfo4: EventsInfo = {
-  eventName: "The Crucible",
-  eventDescription: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
-  eventURL: "https://upload.wikimedia.org/wikipedia/en/7/75/Cruciblecover.jpg",
-  eventFullName: "The Crucible Playbill The Crucible Description Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua."
+  eventName: 'The Crucible',
+  eventDescription: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.',
+  eventURL: 'https://upload.wikimedia.org/wikipedia/en/7/75/Cruciblecover.jpg',
+  eventFullName: 'The Crucible Playbill The Crucible Description Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.',
+};
```

Comment:
It might be a good idea to have a different event from the seed data so we can distinguish them in case one ends up being left around in dev

Reply:
good idea!

Reply:
sawyers next ticket will be to add uuid's to all test data for uniqueness. but henry's suggestion will indeed help troubleshoot any left over test data if they occur.

Reply:
This is actually my next project.  I plan to convert the event interface into a class and give it the ability to append a UID to the event name.